sjb/actions/pull_request_sync.py

```diff
@@ -30,7 +30,7 @@
           <defaultValue></defaultValue>
         </hudson.model.StringParameterDefinition>""")
 
-_PR_SYNC_TITLE_TEMPLATE = Template("SYNC {{ repository | upper }} PULL REQUEST ${{ '{' }}{{ repository | replace('-', '_') | upper }}_PULL_ID}")
+_PR_SYNC_TITLE_TEMPLATE = Template("SYNC {{ repository | upper }} PULL REQUEST {% if len(env['PULL_REFS']) > 0 %}$PULL_REFS{% else %}${{ '{' }}{{ repository | replace('-', '_') | upper }}_PULL_ID}{% endif %}")
```

Comment:
This doesn't seem to work because it is statically generated prior to pushing in Jenkins - @stevekuznetsov not sure how to achieve what I am trying to do here, ie. output the correct variable that is used during the stage execution.